files/task_helper.ps1

```diff
@@ -0,0 +1,261 @@
+# pull some more reusable bits from here
+# https://github.com/puppetlabs/puppetlabs-bootstrap/blob/master/tasks/windows.ps1
+Add-Type -AssemblyName System.ServiceModel.Web, System.Runtime.Serialization
+
+function ConvertTo-JsonString($string)
```

Comment:
this is the only function to keep from this file. the rest were shipped as part of bolt until 3.0, where they were removed when support for powershell 2 was removed